labs/CiscoContainerPlatform-101/2.md

```diff
@@ -13,6 +13,39 @@ Notice in the Cluster details page, a VIP has already been assigned to the Maste
 Each Cisco Container platform installation comes with integrated Open API documentation, powered by OpenAPI (Swagger tooling).
 You can access this [HERE (https://10.10.20.110/2/swaggerapi/)](https://10.10.20.110/2/swaggerapi/), where you see all the currently supported platform APIs.
 
+Note: There is a known issue that the Swagger API page may fail to load.
```

Comment:
See comment about notes.

Suggested rewrite: "Under some circumstances, the Swagger API page may fail to load. This is a known issue. If you are using the Chrome browser, use the following workaround procedure:"

Reply:
Updated 2.md